docs/assistant/azure-openai-setup.asciidoc

```diff
@@ -0,0 +1,131 @@
+[[assistant-connect-to-azure-openai]]
+= Connect to Azure OpenAI
+
+This page provides step-by-step instructions for setting up an Azure OpenAI connector for the first time. This connector type enables you to leverage large language models (LLMs) within {kib}. You'll first need to configure Azure, then configure the connector in {kib}.
+
+[discrete]
+== Configure Azure
+
+[discrete]
+=== Configure a deployment
+
+First, set up an Azure OpenAI deployment:
+
+. Log in to the Azure console and search for Azure OpenAI.
+. In **Azure AI services**, select **Create**.
+. For the **Project Details**, select your subscription and resource group. If you don't have a resource group, select **Create new** to make one.
+. For **Instance Details**, select the desired region and specify a name, such as `example-deployment-openai`.
+. Select the **Standard** pricing tier, then click **Next**.
+. Configure your network settings, click **Next**, optionally add tags, then click **Next**.
+. Review your deployment settings, then click **Create**. When complete, select **Go to resources**.
+
+The following video demonstrates these steps.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/7NEa5VkVJ67RHWBuK8qMXA.jpg"
+  data-uuid="7NEa5VkVJ67RHWBuK8qMXA"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+[discrete]
+=== Configure keys
+
+Next, create access keys for the deployment:
+
+. From within your Azure OpenAI deployment, select **Click here to manage keys**.
+. Store these keys in a secure location. You'll need them later.
+
+The following video demonstrates these steps.
+
+=======
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
+  style="width: 100%; margin: auto; display: block;"
+  class="vidyard-player-embed"
+  src="https://play.vidyard.com/cQXw96XjaeF4RiB3V4EyTT.jpg"
+  data-uuid="cQXw96XjaeF4RiB3V4EyTT"
+  data-v="4"
+  data-type="inline"
+/>
+</br>
+++++
+=======
+
+[discrete]
+=== Configure a model
+
+Now, set up the Azure OpenAI model:
+
+. From within your Azure OpenAI deployment, select **Model deployments**, then click **Manage deployments**.
+. On the **Deployments** page, select **Create new deployment**.
+. Under **Select a model**, choose `gpt-4` or `gpt-4-32k`.
+.. If you select `gpt-4`, set the **Model version** to `0125-Preview`. 
```

Comment:
Minor edit here because these aren't sub-steps. Uers can choose either option.
```suggestion
** If you select `gpt-4`, set the **Model version** to `0125-Preview`. 
```